### PR TITLE
LPD-15680_move_k8s_out_of_static

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi Tina,

Resent: https://github.com/liferay-core-infra/liferay-portal/pull/6551
In order to do deeper jar clean up and centralization, I am trying to move portal-k8s-agent out of static as a preparation.
Thanks for checking

### CODE REVIEW

**Comment by jeffwu0724** [comment]
ci:test:relevant

**Comment by jeffwu0724** [comment]
ci:test

**Comment by jeffwu0724** [comment]
ci:test

**Comment by jeffwu0724** [comment]
ci:test:sf

**Comment by jeffwu0724** [comment]
ci:test:relevant

**Comment by jeffwu0724** [comment]
ci:report:2533698387

**Comment by jeffwu0724** [comment]
no unique failure found in relevant tests, waiting for defualt test result

**Comment by jeffwu0724** [comment]
ci:report:2533813871

**Comment by jeffwu0724** [comment]
ci:report:2533813871

**Comment by jeffwu0724** [comment]
https://testray.liferay.com/home/-/testray/case_results/2533823635/history
https://testray.liferay.com/home/-/testray/case_results/2533822310/history
https://testray.liferay.com/home/-/testray/case_results/2533823282/history

**Comment by jeffwu0724** [comment]
ci:test:sf

**Comment by jeffwu0724** [comment]
ci:test:relevant

**Comment by jeffwu0724** [comment]
forced push and rerun test

**Comment by jeffwu0724** [comment]
ci:report:2536770528

**Comment by jeffwu0724** [comment]
ci:stop

**Comment by jeffwu0724** [comment]
ci:report:2536770528

**Comment by jeffwu0724** [comment]
I have run defualt test before, and they passed my review. 
As I only fixed the code conflict in bnd this time, I think running relevant test should be good enough. 
The only unique regression is https://testray.liferay.com/home/-/testray/case_results/2536773943/history, but it is also in [upstream ](https://testray.liferay.com/home/-/testray/case_results/2537427268?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcases%2F1518352892%2Fview%3Fcur%3D1%26delta%3D200%26testrayCaseId%3D1518352892%26orderByCol%3DcreateDate_sortable%26orderByType%3Ddesc)
So I will mark this passed

File: modules/apps/portal-k8s-agent/test.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+##
+## Testray
+##
+
+    testray.main.component.name=Tools
```
Comment (jeffwu0724) [inline modules/apps/portal-k8s-agent/test.properties:5]: Talked with Victor and he thinks using "Tools" will be the best choice here

**Comment by jeffwu0724** [comment]
Hi @tinatian ,

Could you please help me review this PR? If this PR get merged, then I will have around 2-3 PRs related to "not directly deploying to osgi/portal" can be moved forward.

Thanks!

**Comment by tinatian** [comment]
@jeffwu0724 As we talked, I am going to close this pull, please resend if necessary.

Thanks.
Tina

**Pull request closed without merging**